packages/alea-frontend/pages/index.tsx

```diff
@@ -282,12 +284,29 @@ function AleaFeatures({ img_url, title, description }) {
 
 const StudentHomePage: NextPage = ({ filteredCourses }: { filteredCourses: CourseInfo[] }) => {
   const router = useRouter();
+  const [isInstructor, setIsInstructor] = useState(false);
```

Comment:
dont use `useState`. isInstructor can be derived from resourcesForInstructor directly at render time.